main.tf

```diff
@@ -13,6 +24,15 @@ resource "aws_sfn_state_machine" "this" {
   role_arn   = var.use_existing_role ? var.role_arn : aws_iam_role.this[0].arn
   definition = var.definition
 
+  dynamic "logging_configuration" {
+    for_each = var.logging_configuration.level == "OFF" ? [] : list(var.logging_configuration)
```

Comment:
User may want to bring another cloudwatch_log_group created outside of this module. Please make required changes (see https://github.com/terraform-aws-modules/terraform-aws-lambda module for reference).

Reply:
`list()` => `[]`

Reply:
> User may want to bring another cloudwatch_log_group created outside of this module. Please make required changes (see https://github.com/terraform-aws-modules/terraform-aws-lambda module for reference).

Sorry, not sure I understand this. You want me to make `aws_cloudwatch_log_group` a separate module?